tensorflow/compiler/tests/random_ops_test.py

```diff
@@ -116,12 +116,22 @@ def testTruncatedNormalIsNotConstant(self):
     def rng(dtype):
       return random_ops.truncated_normal(shape=[2], dtype=dtype)
 
-    self._testRngIsNotConstant(rng, dtypes.float32)
+    # There is no double precision float kernel for truncated normal
+    for dtype in self._random_types() & self.float_types - {np.float64}:
+      self._testRngIsNotConstant(rng, dtype)
 
   def testTruncatedNormalIsInRange(self):
-    count = 10000000
-    # TODO(b/34339814): make this test work with 16 bit float types.
-    for dtype in self._random_types() & {dtypes.float32, dtypes.float64}:
+    count = 1000000
```

Comment:
note: numpy gives an error with 10,000,000 reduction during the tests at the end. this count seems to provide an adequate level of accuracy.

Reply:
Can you be more specific -- what's the error?  And can you please add this as a comment?